components/var_exporter.rst

```diff
@@ -274,6 +274,12 @@ While you never query ``$processor->hash`` value, heavy methods will never be
 triggered. But still, the ``$processor`` object exists and can be used in your
 code, passed to methods, functions, etc.
 
+.. deprecated:: 6.4
+
+   The use of ``propertyName`` and ``propertyScope`` in the initializer function
+   is deprecated since Symfony 6.4 and removed in Symfony 7.0.
```

Comment:
```suggestion
   is deprecated since Symfony 6.4 and will be removed in Symfony 7.0.
```

Reply:
lets move this below the paragraph, thanks

Reply:
Done!